src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -124,13 +130,185 @@ Status RedisStoreClient::AsyncDelete(const std::string &table_name,
   return shard_context->RunArgvAsync(args, delete_callback);
 }
 
+Status RedisStoreClient::AsyncBatchDelete(const std::string &table_name,
+                                          const std::vector<std::string> &keys,
+                                          const StatusCallback &callback) {
+  auto finished_count = std::make_shared<int>(0);
+  int size = keys.size();
+  for (auto &key : keys) {
```

Comment:
No need to loop here. Redis supports deleting multiple keys at once.

Reply:
fixed